pkg/static/login.js

```diff
@@ -888,7 +888,9 @@
                     }
                 }
             } else if (xhr.status == 403) {
-                login_failure(_(decodeURIComponent(xhr.statusText)) || _("Permission denied"));
+                host_failure(_(decodeURIComponent(xhr.statusText)) || _("Permission denied"));
```

Comment:
commit message does not explain why you moved this from `login_failure` to `host_failure`.

Reply:
Yes hmm and neither do I... switched it back.